collect_app/src/test/java/org/odk/collect/android/widgets/ExIntegerWidgetTest.java

```diff
@@ -34,4 +37,17 @@ public void setUp() throws Exception {
     private int randomInteger() {
         return Math.abs(random.nextInt()) % 1_000_000_000;
     }
+
+    @Test
+    public void digitsAboveLimitOfNineShouldBeTruncatedFromLeft() {
```

Comment:
This might be a "swipe left"/"swipe right" ambiguity problem but should these not all be `TruncatedFromRight`?

Reply:
Good catch it's a test I copied from another one and didn't notice it.